specs/experimental/scan_page_meta.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { test, expect } from 'vitest'
+import { fileURLToPath } from 'node:url'
+import { setup, url } from '../utils'
+import { getText, renderPage } from '../helper'
+
+await setup({
+  rootDir: fileURLToPath(new URL(`../fixtures/basic_usage`, import.meta.url)),
+  browser: true,
+  // overrides
+  nuxtConfig: {
+    experimental: {
+      scanPageMeta: true
```

Comment:
I think nuxt i18n should check the `scanPageMeta` value and warn if it is `false`.
Because, if that value changes to `false`, nuxt i18n route aliases don't work.

Reply:
I have just added a warning that logs when the feature is not enabled [`8cab851` (#2962)](https://github.com/nuxt-modules/i18n/pull/2962/commits/8cab8511c5e597dbfe4bceb4c79875ad392100c5)!